discussion/views.py

```diff
@@ -185,6 +188,17 @@ def create_attachments(request):
     # The user is uploading one or more files.
     ret = { }
     for fn in request.FILES:
+        # Validate before attachment object creation
+        uploaded_file = request.FILES[fn]
+
+        # 2.5MB
```

Comment:
I think 2.5MB is going to be too small. I think we need to add a different story where the size of upload is set in the data mode `system_setting`.

Reply:
Most images are KBs but there might be an image/pdf file that is in the MBs. Makes sense will make another issue.

Reply:
Sounds good.

Reply:
@gregelin here's the related issue https://github.com/GovReady/govready-q/issues/1078